composer.json

```diff
@@ -34,22 +34,39 @@
         "twig/twig" : "2.7.4",
         "vlucas/phpdotenv" : "3.3.3",
         "yubico/u2flib-server" : "1.0.2",
-        "zendframework/zendframework" : "2.5.3",
+        "zendframework/zend-db" : "2.10.0",
+        "zendframework/zend-mvc" : "3.1.1",
+        "zendframework/zend-loader" : "2.6.0",
+        "zendframework/zend-inputfilter" : "2.10.0",
+        "zendframework/zend-soap" : "2.7.0",
+        "zendframework/zend-servicemanager" : "3.4.0",
+        "zendframework/zend-eventmanager" : "3.2.1",
+        "zendframework/zend-stdlib" : "3.2.1",
+        "zendframework/zend-escaper" : "2.6.0",
+        "zendframework/zend-config" : "3.1.0",
+        "zendframework/zend-filter" : "2.9.1",
+        "zendframework/zend-form" : "2.14.1",
+        "zendframework/zend-xmlrpc" : "2.7.0",
+        "zendframework/zend-session" : "2.8.5",
+        "zendframework/zend-json" : "3.1.0",
         "particle/validator": "2.3.4",
         "guzzlehttp/guzzle": "6.3.3",
         "kamermans/guzzle-oauth2-subscriber": "1.0.6",
         "league/omnipay": "3.0.2",
         "academe/omnipay-authorizenetapi": "3.1.1",
         "omnipay/stripe": "3.0.1",
         "waryway/php-traits-library": "1.0.4",
-        "paragonie/multi-factor": "0.2.2"
+        "paragonie/multi-factor": "0.2.2",
+        "zendframework/zend-mvc-i18n": "1.1.0",
+        "zendframework/zend-json-server": "3.1.0"
     },
     "config": {
         "platform": {
             "php": "7.1"
         }
     },
     "require-dev" : {
+        "zendframework/zend-component-installer": "^2.1"
```

Comment:
Is this needed?

Reply:
Zend Framework 3 recommends it be installed as it will adjust the composer.json file as you install components in the way zend recommends it.  I haven't seen it do much as its also supposed to auto update our application.config.php file.  However, that file is a different spot then zend expects it so it can't update it.  I think we should be safe removing this.  Good catch.